### PR TITLE
Update Google Drive  (Allow subfolders for Google Drive mounts)

### PR DESCRIPTION
**Pull request by mmattel**
This PR updates the Google Drive document
It documents the changes made in core: https://github.com/owncloud/core/pull/38161 (Allow subfolders for Google Drive mounts).
In addition, the document flow has been reviewed and improved.

Refs: https://github.com/owncloud/docs/issues/2887

I ran an antora build and it looks great.

DO NOT MERGE until a 10.6 branch has been created in docs. It is atm not sure if the change in core will be part of the 10.6 release. When the 10.6 branch has been created in docs, we can merge securely and backport if needed.

@pmaier1 fyi

### CODE REVIEW

File: modules/admin_manual/pages/configuration/files/external_storage/google.adoc [hunk 1]
```diff
@@ -1,131 +1,194 @@
 = Google Drive
+:toc: right
+:google_cloud_console: https://console.developers.google.com/
 
+== Introduction
+
+Using the Google Drive external storage in ownCloud, you can mount all or 
+a subfolder of Google Drive.
+
+For subfolders, use following scheme:
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/files/external_storage/google.adoc:10]:
```suggestion
For subfolders, use the following scheme:
```

File: modules/admin_manual/pages/configuration/files/external_storage/google.adoc [hunk 2]
```diff
@@ -1,131 +1,194 @@
...
+
+- subfolder = empty (mounting the root, all of Google Drive will be used)
+- subfolder = $user ($user variable represents the current logged in ownCloud username)
+- subfolder = name (a folder name, can be cascaded like name1/name2 or name/$user)
+
+NOTE: If the subfolder is not present in Google Drive, no Google Drive mount will
+be shown in the users file list.
+
+NOTE: Using subfolders is benificial, if you want to selectively encrypt Googler Drive mount points
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/files/external_storage/google.adoc:19]:
```suggestion
NOTE: Using subfolders is benificial, if you want to selectively encrypt Google Drive mount points
```

File: modules/admin_manual/pages/configuration/files/external_storage/google.adoc [hunk 3]
```diff
@@ -1,131 +1,194 @@
...
+
+----
+https://example.com
+or
+http://example.com
+or
+IP/owncloud  
+----
+
+== Preperations in the Google Cloud Console
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/files/external_storage/google.adoc:43]:
```suggestion
== Preparations in the Google Cloud Console
```

File: modules/admin_manual/pages/configuration/files/external_storage/google.adoc [hunk 4]
```diff
@@ -1,131 +1,194 @@
...
++
 image:configuration/files/external_storage/google_drive/004.png[Notification bell]
 
-Go to Api overview to select google’s API.
-
+. Go to Api overview to select google’s API.
++
 image:configuration/files/external_storage/google_drive/005.png[API]
 
+. Select Goolge Drive API
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/files/external_storage/google.adoc:73]:
```suggestion
. Select Google Drive API
```

File: modules/admin_manual/pages/configuration/files/external_storage/google.adoc [hunk 5]
```diff
@@ -1,131 +1,194 @@
...
 
+. Enable external Storage
++
 image:configuration/files/external_storage/google_drive/013.png[Enable external Storage]
 
+. Select Google Drive
++
 image:configuration/files/external_storage/google_drive/014.png[Select Google Drive from dropdown menu]
 
+. Enabled Google Drive App
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/files/external_storage/google.adoc:137]:
```suggestion
. The Google Drive App is enabled
```

File: modules/admin_manual/pages/configuration/files/external_storage/google.adoc [hunk 6]
```diff
@@ -1,131 +1,194 @@
...
++
+Give the mount point a meaningful name. We have used `Google Drive` in this example.
++
 image:configuration/files/external_storage/google_drive/015.png[Now you have your Google Drive App enabled]
 
+. Copy from the browser the Authorized Redirect URI
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/files/external_storage/google.adoc:143]:
```suggestion
. Copy the Authorized Redirect URI from the browser
```

**Comment by mmattel** [comment]
@phil-davis thanks for reviewing, all suggestions committed.

**Review by micbar — APPROVED** [review]
THX

**Comment by mmattel** [comment]
The 10.6 branch has been created, ready to merge.
@phil-davis can you do so?

File: modules/admin_manual/pages/configuration/files/external_storage/google.adoc [hunk 7]
```diff
@@ -1,131 +1,193 @@
...
-`\https://example.com`, without a trailing slash. Examples:
+=== Configure Authorisations
 
-----
-https://example.com
-http://example.com
-IP/owncloud  
-----
+These authorisations are necessary to tell Google which source URI requests are allowed.
+You can configure multiple Authorized URIs if you whish to enable admin and personal
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/files/external_storage/google.adoc:96]:
```suggestion
You can configure multiple Authorized URIs if you wish to enable admin and personal
```

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
This can be merged when CI finishes.
No backports are required. The subfolders functionality will come in a core release after 10.6.0.

**Comment by mmattel** [comment]
Finally, "All checks have passed" :smile:

**Pull request merged**